qualityattenuation/qualityattenuation.go

```diff
@@ -0,0 +1,133 @@
+// Implements a data structure for quality attenuation.
+
+package qualityattenuation
+
+import (
+	"math"
+
+	"github.com/influxdata/tdigest"
+)
+
+type SimpleQualityAttenuation struct {
+	EmpiricalDistribution     *tdigest.TDigest
```

Comment:
In go when you start a member name with a capital letter that makes it public. Because you have done the great work of making nice *getters*, I don't think that we need to have these as public. 

(see below)